source/uwp/Renderer/lib/AdaptiveTextBlock.cpp

```diff
@@ -190,8 +190,12 @@ AdaptiveNamespaceStart
         textBlock->SetText(text);
 
         std::string language;
-        RETURN_IF_FAILED(HStringToUTF8(m_language.Get(), language));
-        textBlock->SetLanguage(language);
+        //  If the language was not specified the conversion fails, just keep the language as an empty string
+        if (!(WindowsIsStringEmpty(m_language.Get())))
+        {
+            RETURN_IF_FAILED(HStringToUTF8(m_language.Get(), language));
```

Comment:
This seems like an unrelated change. Can you make it as a separate pull request and issue?